plugins/kotlin/j2k/shared/tests/testData/newJ2k/liftReturns/tryCatchStatement.kt

```diff
@@ -0,0 +1,9 @@
+internal class Test {
+    fun test(): String {
+        return try {
```

Comment:
This is questionable from a code style perspective. I believe the inspection doesn't highlight such code as a warning, so it should not be automatically converted by J2K as well. I think lifting the return out of `try` only makes sense if both `try` and `catch` blocks contain exactly one return statement and nothing else.